src/main/java/com/minecolonies/coremod/colony/colonyEvents/raidEvents/AbstractShipRaidEvent.java

```diff
@@ -117,6 +120,16 @@
      */
     private int shipRotation = 0;
 
+    /**
+     * List of all spawners.
+     */
+    private List<BlockPos> spawners = new ArrayList<>();
+
+    /**
+     * Count of spawners.
+     */
+    private int spawnerCount;
```

Comment:
is this int even still needed anymore? think you can remove it. Seems like it isnt getting data anymore aswell so logic on it might break

Reply:
Backwards compatibility. For ships that are still in some worlds.

Reply:
ah ok